src/pages/portfolio/index.tsx

```diff
@@ -132,71 +79,54 @@ const PortfolioPage: FC<{ data: ProjectGQL }> = ({ data }) => {
   };
 
   return (
-    <Container className={classes.container} maxWidth="lg">
-      <Helmet>
-        <title>{projectData.portfolioPageTitle}</title>
-        <meta name="description" content={projectData.portfolioPageDescription} />
-        <meta property="og:title" content={projectData.portfolioPageTitle} />
-        <meta property="og:description" content={projectData.portfolioPageDescription} />
-        <meta property="og:image" content={projectData.portfolioPageImage.publicURL} />
-      </Helmet>
-      {isDesktop && (
-        <Box className={classes.aside}>
-          <DetailsCard type={componentType} />
-        </Box>
-      )}
-      <Box className={classes.main}>
-        <Navbar
-          className={classes.navbar}
-          type={componentType}
-          fullName={`${developerProfile.firstName} ${developerProfile.lastName}`}
-          position={developerProfile.position}
-          image={developerProfile.avatar.publicURL}
-          resumeLink={developerProfile.cv}
-        />
-        <Box className={classes.mainContent}>
-          <Box className={classes.projectsContainer}>
-            <SectionTitle className={classes.title}>My works</SectionTitle>
-            <Box className={classes.projects}>
-              {projectData.projects.map((project, index) => (
-                <div key={`${project.projectName}-${project.projectDescription}`}>
-                  <PortfolioCard
-                    className={classes.project}
-                    type={componentType}
-                    title={project.projectName}
-                    label={project.projectLabel}
-                    description={project.projectDescription}
-                    image={project.projectPreviewImage.publicURL}
-                    onClick={() => setSelectedProject(index)}
-                  />
-                  <PortfolioProjectDialog
-                    type={componentType}
-                    handleClose={() => handleCloseProject()}
-                    handlePrev={() => handlePreviousProject(index)}
-                    handleNext={() => handleNextProject(index)}
-                    isOpen={index === selectedProject}
-                    title={project.projectName}
-                    tags={project.projectTechnologies.map((technology) => ({ name: technology.technologyName }))}
-                    imgurl={project.projectPreviewImage.publicURL}
-                    subtitle={`
+    <Layout
+      developerProfile={developerProfile}
+      meta={{
+        title: projectData.portfolioPageTitle,
+        description: projectData.portfolioPageDescription,
+        imageUrl: projectData.portfolioPageImage.publicURL,
+      }}
+    >
+      <Box className={classes.projectsContainer}>
+        <SectionTitle className={classes.title}>My works</SectionTitle>
+        <Box className={classes.projects}>
+          {projectData.projects.map((project, index) => (
```

Comment:
Move callback of this map to a separate function

Reply:
Good idea, I will do this after [THIS PR](https://github.com/CodersCrew/CodersCard/pull/129) #129  is merged 😄